### DISCUSSION TITLE
Is there a way to use tensorflows prefetch functionality with minio buckets?

### DISCUSSION THREAD

**Discussion by johanneszellinger** [minio/minio#14521] [Q&A] [answered]
I'm wondering if there is the possibility to use [tf.data.Dataset.prefetch()](https://www.tensorflow.org/api_docs/python/tf/data/Dataset#prefetch) in a python script, which access its data from a bucket of a minio docker container?

This would increase performance quite a bit instead of having to load each file one by one....

**Comment by harshavardhana** [comment, marked as answer]
Not the right place to ask this question @TheRealRolandDeschain please ask in tensorflow forums.
